lib/tests/unit/test_plugin.py

```diff
@@ -154,8 +153,16 @@ def check_tc(params, expected):
 
 
 def enrich_params_with_mandatory_keys(params):
-    connection_parameters = {'session_id': '', 'protocol': '', 'connection_name': '', 'client_ip': '', 'client_port': '', 'gateway_user': '',
-     'target_username': '', 'key_value_pairs': {}, 'cookie': {}, 'session_cookie': {}}
+    connection_parameters = {'session_id': '',
```

Comment:
the enclosing { and } should be on its own line. So session_id should be on a new line , and the rest aligned with session_id:
connection_parameters = {
    'session_id': '',
    ....
    'session_cookie': {},
}

Reply:
I fixed it.